README.md

```diff
@@ -51,6 +51,42 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
+1. Requirements:
+
+   1. Clone this repo.
```

Comment:
I don't think it's possible to do such thing `docker build link` but I know it's possible to register this image on [docker hub register](https://registry.hub.docker.com/).

You're right, we can avoid to have this file (Dockerfile) on repo by providing only the instructions:
```Shell
docker run --name edash -d edash
```

I'm gonna see how to upload the image and use this register.